app/assets/stylesheets/buttons.scss

```diff
@@ -1,4 +1,29 @@
 input[type="submit"] {
-  font-size: 30px;
+  font-size: px;
 
 }
+button {
+  color: #fff;
+  background-color: $green-dark;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0 5px $green-dark2;
+  padding: 5px 10px 5px 10px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `5px 10px` instead of `5px 10px 5px 10px`